### DISCUSSION TITLE
How to clear the context menu item's

### DISCUSSION THREAD

**Discussion by jddcef** [AutomaApp/automa#844] [General]
Hi, my context menu has a bunch of duplicates 
![image](https://user-images.githubusercontent.com/608676/195334832-25264686-db30-4f1a-bdac-351f875f624d.png)
How do I reset it to start over?
Is there no place I can see all the context menu options that Automa is registering? Like a global manager?

**Comment by Kholid060** [comment]
Open the Automa dashboard, press shift+key+i, paste the below code, and press Enter key.
```js
chrome.contextMenus.removeAll()
```

**Comment by jddcef** [comment]
`ctrl-shift-i`
thank you
